src/components/InputItem/InputItem.js

```diff
@@ -0,0 +1 @@
+InputItem
```

Comment:
Codacy found an issue: ['InputItem' is not defined.](https://app.codacy.com/gh/WebHeroSchool/evafinal/pullRequest?prid=5874309)